src/libraries/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -14,6 +14,8 @@
   </PropertyGroup>
   <ItemGroup Condition="'$(TargetsBrowser)' != 'true'">
     <!-- All configurations -->
+    <Compile Include="System\Net\Connections\SocketsConnectionFactory.cs" />
```

Comment:
I thought this class was intended to live in System.Net.Connections, per comments on #40044. Did I misunderstand?

Reply:
My interpretation was that the comment was about the namespace, not the dll.

@scalablecory what dependency order do we want?
1. `Connections` --> `Sockets` OR
2. `Sockets` --> `Connections`

For me it seemed kinda obvious that 2. is better, since `System.Net.Connections` is intended to contain abstractions, not implementations. Was I wrong?

Reply:
I don't know the intent here. But it seems weird to have the namespace be `System.Net.Connections`, but it lives in `System.Net.Sockets`.

Reply:
If we put the socket factory in System.Net.Connections it does make it easier to make a nuget package.

@DavidFowl previously wanted such a package for SignalR.

Other than that I think it's probably okay to put it in System.Net.Sockets, though making Pipelines a dependency is not great.

Reply:
Looks like 3 counterarguments against `System.Net.Sockets`, I will move stuff then I guess.